src/main/kotlin/no/nav/familie/ef/sak/vedtak/VedtakController.kt

```diff
@@ -69,6 +73,20 @@ class VedtakController(private val stegService: StegService,
         return Ressurs.success(vedtakService.hentVedtakHvisEksisterer(behandlingId))
     }
 
+    @PostMapping("/{behandlingId}/fullfor")
```

Comment:
Trenger vi `fullfor` her?

Reply:
det går å ha uten, men er det ikke ryddigst for lesbarheten sin del at vi har med fullfor i url? Var lett å finne frem i frontend koden også med den.